angular-workspace/projects/ni/nimble-angular/src/directives/radio/radio-control-value-accessor.directive.ts

```diff
@@ -0,0 +1,297 @@
+/* eslint-disable @angular-eslint/no-host-metadata-property */
+/* eslint-disable @angular-eslint/directive-class-suffix */
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable @typescript-eslint/member-ordering */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-member-accessibility */
+/* eslint-disable max-classes-per-file */
+
+/**
+ * The contents of this file were taken with minimal modification from Angular's source:
```

Comment:
Guess it wasn't discussed how we should approach this it but I expect us to probably need to copy more of these until [we convince Angular to make these public](https://github.com/ni/nimble/issues/732#issuecomment-1245814588) and I want these to be as easy as possible to diff for changes with Angular. Placing three separate files into one seems like it would make that harder.

Can we instead create with the file names unchanged:
```
/nimble-angular/src/thirdparty/directives/
ng_control.ts
control_value_accessor.ts
radio_control_value_accessor.ts
```

You can also disable linting in that whole folder using an `.eslintrc.js` file. We should not apply out own linting at all to make diffs from the original less noisy

Also within the individual files the top comment it should describe more precisely what changes are made (imports updated, certain classes removed because they are public, etc).

Reply:
I replaced the one file with multiple files, as you suggested. The files have almost no modifications, and what they do have is documented (as you suggested). I excluded the `thirdparty` directory from linting in one of the existing `.eslintrc.js` files.